geode-protobuf/src/main/proto/clientProtocol.proto

```diff
@@ -66,7 +66,7 @@ message Response {
         RemoveAllResponse removeAllResponse = 7;
         ListKeysResponse listKeysResponse = 8;
 
-        ErrorResponse errorResponse = 13;
+        Error error = 13;
```

Comment:
Why has this been renamed to Error from ErrorResponse? Imo, we are following the pattern (good or bad) that a ClientProtocol.Response contains an Operation specific response. To now have a non-"Response" message sort of breaks the mold.